sunpy/map/mapbase.py

```diff
@@ -79,33 +79,15 @@
 corresponds to the coordinate axis for ``x`` and ``axis2`` corresponds to
 ``y``.
 
-This class makes some assumptions about the WCS information contained in
-the meta data. The first and most extensive assumption is that it is
-FITS-like WCS information as defined in the FITS WCS papers.
-
-Within this scope it also makes some other assumptions.
-
-* In the case of APIS convention headers where the CROTAi/j arguments are
-    provided it assumes that these can be converted to the standard PCi_j
-    notation using equations 32 in Thompson (2006).
-
-* If a CDi_j matrix is provided it is assumed that it can be converted to a
```

Comment:
This isn't an assumption we make, it's just the definition of the relation between CD_ij, CDELTi, and PC_ij, so I've dropped this bullet.